src/pages/docs/kv/api/delete-batch.mdx

```diff
@@ -3,6 +3,8 @@ import { CodeGroup, Code } from '@/components/Code';
 
 # deleteBatch
 
+<Warning>This is an experimental feature - stability of the API is not guaranteed. Proceed with caution.</Warning>
```

Comment:
Can we instead add this as a templated message in the `src/components/mdx.jsx` file? That way it's just a bit easier to change if needed :)